amanuensis/blueprints/project.py

```diff
@@ -43,35 +66,32 @@ def get_projetcs():
         tmp_project["id"] = project["id"]
         tmp_project["name"] = project["name"]
 
-        status_code = None
-        status = None
+        requests_status_codes = []
+        request_status_code = None
+        project_status_code = None
+        project_status_name = None
         status_date = None
         submitted_at = None
         completed_at = None
+
         for request in project["requests"]:
-            # get status
-            # "status": "", // string ("IN REVIEW" | "APPROVED" | "REJECTED")
-            # "submitted_at": "", // string (timestamp) or null
-            # "completed_at": "" // string (timestamp) or null
+            request_state = current_session.query(RequestState).filter(RequestState.request_id == request["id"]).all()[-1]
+            request_status_code = STATE_ID_TO_CODE[request_state.state_id]
+            requests_status_codes.append(request_status_code)
```

Comment:
Is there a reason why you are not just using the ORM to get the states associated to the request?
request_state.state.code